README.md

```diff
@@ -1,138 +1,126 @@
-  * ~~Set issue check files to be encrypted, so that images can be released but competitors can not find out what "issues" need to be fixed~~
-  * ~~Add files to the relevant Operating System startup folders, so that CSSS will start automatically:~~
-    * ~~[Windows shortcut](http://stackoverflow.com/a/19914018) in [Programdata startup folder](https://www.kiloroot.com/all-users-or-common-startup-folder-locations-launch-programs-at-window-login-windows-server-2008-r2-2012-2012-r2/)~~
-    * ~~[Linux](http://raspberrypi.stackexchange.com/a/5159)~~
-  * Email list of issues to be fixed when '--email' argument is passed
-  * Shutdown the computer to allow an image to be taken
-* Comment ~~and coding style~~ tidy-up
-
-### Version 0.4
-* Allow creating "named" images, so that it's possible to keep track of multiple images
-* Enter team identifiers to keep track of who is using which image
-* A server instance that will provide:
-  * A central store of scoring reports for a team and their images in that "round"
-  * Scoreboards and graphs to show which team is in the lead
-  * A way to send messages from the server to all images (e.g. "Time limit is approaching")
-  * Checks to make sure that only one instance of an image is open at once
-
-### Version 0.5
-* Create scripts to install CSSS files into the correct locations (WinNT: root of `%SystemDrive%`, Linux: `/opt` directory)
-* Install additional software when CSSS is being installed to make sure all items are available for it to use
-* Create build scripts that install software that can be used when testing the checker system engine
-
-### Versions 0.6 onwards
-* Create more check template files and relevant classes for the checker system engine to use (a new version for each check "category")
-* Create some example checker files with instructions to create images based around them with pre-made questions and scenarios
-
-## Copyright
-CyberPatriot is copyright of US Air Force Association
-
-CyberCentrurion is copyright of Cyber Security Challenge UK
-
-## Disclaimer
-This program is designed to score the state of a computer image against a pre-defined checklist of "issues". It is not designed to rate your computer security, provide security advice, or anything else that isn't related to the CyberPatriot / CyberCenturion competitions.
+# CyberSecurity Scoring System (CSSS)
+
+[![.Net Versions](https://img.shields.io/badge/.Net%20Versions-4.0|4.5|4.6|4.7|4.8-blue.svg)]()
+[![Status](https://img.shields.io/badge/status-Pre%20Release-red.svg)]()
+[![Releases](https://img.shields.io/github/v/release/stuajnht/CSSS?include_prereleases)]()
+[![Maintenance](https://img.shields.io/maintenance/yes/2019.svg)]()
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v1.4%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
+[![Known Vulnerabilities](https://snyk.io/test/github/stuajnht/CSSS/badge.svg?targetFile=CSSS.sln)](https://snyk.io/test/github/stuajnht/CSSS?targetFile=CSSS.sln)
```

Comment:
Could you have a look to see if you can get a badge to display for Snyk, as it currently just says "Known Vulnerabilities" when [viewed on GitHub](https://github.com/Stormy102/CSSS/blob/feature/ReadMeBadgify/README.md#cybersecurity-scoring-system-csss). I think it may only work for a [Node.js, Ruby or Java GitHub repository](https://snyk.io/docs/badges/#github-badge).

Reply:
You need to setup Snyk correctly for it to display. I'm not sure if you've done this yet, so I'll comment out the badge for the future.

Reply:
I've added CSSS to my Snyk account. However, it looks like you can't just add a badge direct to the `CSSS.sln` file, but for each project inside it?

For example, Snyk gives me a badge code of:
```  
[![Known Vulnerabilities](https://snyk.io/test/github/stuajnht/CSSS/badge.svg?targetFile=CSSS/CSSS.csproj)](https://snyk.io/test/github/stuajnht/CSSS?targetFile=CSSS/CSSS.csproj)
```

... which renders as:
[![Known Vulnerabilities](https://snyk.io/test/github/stuajnht/CSSS/badge.svg?targetFile=CSSS/CSSS.csproj)](https://snyk.io/test/github/stuajnht/CSSS?targetFile=CSSS/CSSS.csproj)

... meaning that we'd need to have one badge for each project in the solution (which will start to look messy).

However, as mentioned in [my previous comment](https://github.com/stuajnht/CSSS/pull/3#discussion_r338809991), I don't think Snyk badges work for .Net projects (at the moment?). Therefore, it is worth just removing the commented out badge? Unless you can find another way to get it to work?

Reply:
Snyk does have support for .Net projects - however, only the Command Line is supported for .sln - see [here](https://snyk.io/docs/snyk-for-dotnet/)

A work-around would be displaying the badge for the whole repo. However, until the master branch (or the branch you specified for Snyk to monitor) is updated, the badge won't display properly. Therefore I'm going to remove the badge for now.